### PR TITLE
Update Fetch Migration documentation

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
- Edit Fetch Migration Readme to match the AWS style guide.
- Tweak steps to account for issues encountered during testing.

### Description
[Describe what this change achieves]
* Category (Enhancement, New feature, Bug fix, Test fix, Refactoring, Maintenance, Documentation)
* Why these changes are required?
* What is the old behavior before changes and new behavior after changes?

### Issues Resolved
[List any issues this PR will resolve]

Is this a backport? If so, please add backport PR # and/or commits #

### Testing
[Please provide details of testing done: unit testing, integration testing and manual testing]

### Check List
- [ ] New functionality includes testing
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by Naarcha-AWS** [comment]
@kartg: Could you take a look at this one and make sure my changes are accurate? The tool itself worked.

File: FetchMigration/README.md [hunk 1]
```diff
@@ -1,74 +1,76 @@
...
+* Fetch Migration runs as a single instance and does not support vertical scaling or data slicing.
+* Fetch Migration does not support customizing the list of indices included for migration.
+* Fetch migration only supports basic authentication.
+* The migration does not filter out indexes whose health is `red`.
+* In the event that the migration fails or the process dies, the created indexes on the target cluster are not rolled back to the previous version.
+
+## Running Fetch Migration
 
-## Execution
+Use either [Python](#python) or [Docker](#Docker) to run the Fetch Migration tool.
```
Comment (sumobrian) [inline FetchMigration/README.md:33]: This tooling requires python3. We should call that out specifically.

File: FetchMigration/README.md [hunk 2]
```diff
@@ -1,74 +1,76 @@
...
 
 ### Python
 
-* [Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) this GitHub repo
-* Install [Python](https://www.python.org/)
-* Ensure that [pip](https://pip.pypa.io/en/stable/installation/#) is installed
-* (Optional) Set up and activate a [virtual environment](https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-and-using-virtual-environments)
+1. [Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) the `opensearch-migrations` repo using `git clone git@github.com:opensearch-project/opensearch-migrations.git`.
+2. Install [Python](https://www.python.org/).
```
Comment (sumobrian) [inline FetchMigration/README.md:38]: Python3

File: FetchMigration/README.md [hunk 3]
```diff
@@ -1,74 +1,76 @@
...
 
 ### Docker
 
-First build the Docker image from the `Dockerfile`:
+To use the Fetch Migration tool with Docker, use the following steps:
 
-```shell
-docker build -t fetch-migration .
-```
+1. From your cloned Git repository, build the `fetch-migration` Docker image using the f
```
Comment (sumobrian) [inline FetchMigration/README.md:59]: The end of this looks incomplete

**Review by sumobrian — CHANGES_REQUESTED** [review]

File: FetchMigration/README.md [hunk 4]
```diff
@@ -1,74 +1,76 @@
...
-comparing indices between the two clusters and only creates index metadata (settings and mappings) that do not already 
-exist on the target cluster. Internally, the tool uses [Data Prepper](https://github.com/opensearch-project/data-prepper) 
-to migrate data for these created indices.
+The **Fetch Migration** tool simplifies the process of moving indexes and the data contained within them from a
+source cluster, such as Elasticsearch or OpenSearch, to a target OpenSearch cluster. It automates the process of 
+comparing indexes between the two clusters by creating index metadata, like settings and mappings that does not already exist on the target cluster using [Data Prepper](https://github.com/opensearch-project/data-prepper).
 
-The Fetch Migration tool is implemented in Python.
-A Docker image can be built using the included [Dockerfile](./Dockerfile).
+The Fetch Migration tool consists of a Python script. However, a Docker image can be built using the included [Dockerfile](./Dockerfile) included in this repository.
```
Comment (Naarcha-AWS) [inline FetchMigration/README.md:7]:
```suggestion
The Fetch Migration tool consists of a Python script which requires Python 3 or greater. However, a Docker image can be built using the included [Dockerfile](./Dockerfile) included in this repository.
```

File: FetchMigration/README.md (hunk 1 above)
Comment (Naarcha-AWS) [inline FetchMigration/README.md:33]:
```suggestion
Use either [python3](#python) or [Docker](#Docker) to run the Fetch Migration tool.
```

File: FetchMigration/README.md (hunk 2 above)
Comment (Naarcha-AWS) [inline FetchMigration/README.md:38]:
```suggestion
2. Install [python3](https://www.python.org/downloads/).
```

Comment (Naarcha-AWS) [reply FetchMigration/README.md:38]: Isn't `pip` apart of the python3 package?

File: FetchMigration/README.md (hunk 3 above)
Comment (Naarcha-AWS) [inline FetchMigration/README.md:59]:
```suggestion
1. From your cloned Git repository, build the `fetch-migration` Docker image using the following command:
```

**Comment by Naarcha-AWS** [comment]
@sumobrian: Ready for another review.

File: FetchMigration/README.md [hunk 5]
```diff
@@ -1,74 +1,76 @@
...
-comparing indices between the two clusters and only creates index metadata (settings and mappings) that do not already 
-exist on the target cluster. Internally, the tool uses [Data Prepper](https://github.com/opensearch-project/data-prepper) 
-to migrate data for these created indices.
+The **Fetch Migration** tool simplifies the process of moving indexes and the data contained within them from a
+source cluster, such as Elasticsearch or OpenSearch, to a target OpenSearch cluster. It automates the process of 
+comparing indexes between the two clusters by creating index metadata, like settings and mappings that does not already exist on the target cluster using [Data Prepper](https://github.com/opensearch-project/data-prepper).
 
-The Fetch Migration tool is implemented in Python.
-A Docker image can be built using the included [Dockerfile](./Dockerfile).
+The Fetch Migration tool consists of a Python script which requires Python 3 or greater. However, a Docker image can be built using the included [Dockerfile](./Dockerfile) included in this repository.
```
Comment (kartg) [inline FetchMigration/README.md:7]: Nit - "However" seems unnecessary here? It can be removed.

File: FetchMigration/README.md [hunk 6]
```diff
@@ -1,74 +1,76 @@
...
+# Fetch  Migration and Backfill
 
-Fetch Migration provides an easy-to-use tool that simplifies the process of moving indices and their data from a 
-"source" cluster (either Elasticsearch or OpenSearch) to a "target" OpenSearch cluster. It automates the process of 
-comparing indices between the two clusters and only creates index metadata (settings and mappings) that do not already 
-exist on the target cluster. Internally, the tool uses [Data Prepper](https://github.com/opensearch-project/data-prepper) 
-to migrate data for these created indices.
+The **Fetch Migration** tool simplifies the process of moving indexes and the data contained within them from a
+source cluster, such as Elasticsearch or OpenSearch, to a target OpenSearch cluster. It automates the process of 
+comparing indexes between the two clusters by creating index metadata, like settings and mappings that does not already exist on the target cluster using [Data Prepper](https://github.com/opensearch-project/data-prepper).
```
Comment (kartg) [inline FetchMigration/README.md:5]: There are some technical inaccuracies here:
1. Data Prepper is only used for data migration, and not for index comparison and creation as this line implies
2. Indexes are compared _before_ they are created, not _by_ creating index metadata.

File: FetchMigration/README.md [hunk 7]
```diff
@@ -1,74 +1,76 @@
...
-* Fetch Migration runs as a single instance and does not support vertical scaling or data slicing
-* The tool does not support customizing the list of indices included for migration
-* Metadata migration only supports basic auth
-* The migration does not filter out `red` indices
-* In the event that the migration fails or the process dies, the created indices on the target cluster are not rolled back
+The Fetch Migration currently has the following limitations:
+
+* Fetch Migration runs as a single instance and does not support vertical scaling or data slicing.
+* Fetch Migration does not support customizing the list of indices included for migration.
+* Fetch migration only supports basic authentication.
```
Comment (kartg) [inline FetchMigration/README.md:27]: Note - this line should be removed once https://github.com/opensearch-project/opensearch-migrations/pull/394 is merged

File: FetchMigration/README.md [hunk 8]
```diff
@@ -1,74 +1,76 @@
...
 
-* [Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) this GitHub repo
-* Install [Python](https://www.python.org/)
-* Ensure that [pip](https://pip.pypa.io/en/stable/installation/#) is installed
-* (Optional) Set up and activate a [virtual environment](https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-and-using-virtual-environments)
+1. [Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) the `opensearch-migrations` repo using `git clone git@github.com:opensearch-project/opensearch-migrations.git`.
+2. Install [python3](https://www.python.org/downloads/).
+3. Ensure that [pip](https://pip.pypa.io/en/stable/installation/#) by entering `pip --version`.
+4. (Optional) Set up and activate a [virtual environment](https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-and-using-virtual-environments).
+5. Navigate to you cloned GitHub repo. Then, install the required Python dependencies by running the following command:
```
Comment (kartg) [inline FetchMigration/README.md:41]: Nit - "your"

**Comment by kartg** [comment]
@Naarcha-AWS are you available to incorporate comments so we can get this merged in?

**Pull request closed without merging**